pkgs/development/python-modules/ruff-lsp/default.nix

```diff
@@ -42,21 +43,21 @@ buildPythonPackage rec {
   doCheck = stdenv.isDarwin;
 
   nativeCheckInputs = [
-    unittestCheckHook
+    pytestCheckHook
     python-lsp-jsonrpc
     ruff
+    unittestCheckHook
```

Comment:
why keep unittestCheckHook if ruff-lsp switched to pytest?

Reply:
good point, I removed it